utils/python/hsfs_utils.py

```diff
@@ -177,6 +177,18 @@ def import_fg(job_conf: Dict[Any, Any]) -> None:
     fg.insert(df)
 
 
+def feature_monitoring(job_conf: Dict[str, Any]) -> None:
```

Comment:
Can we prepend the method name with a verb?  As per the description, maybe `run_feature_monitoring`,  
 `invoke_feature_monitoring` or `setup_feature_monitoring`?

Reply:
I like run ou compute, I'll put run and update the hopsworks-ee PR accordingly